app/controllers/checkout_controller.rb

```diff
@@ -202,4 +207,28 @@ def redirect_to_paypal_express_form_if_needed
     render json: {path: spree.paypal_express_url(payment_method_id: payment_method.id)}, status: 200
     true
   end
+
+  def construct_saved_card_attributes
+    existing_card_id = params[:order].delete(:existing_card)
+    if existing_card_id.present?
```

Comment:
early return maybe? `return unless existing_card_id.present?` or something like that.

Reply:
Resolved